fid/fid.c

```diff
@@ -161,6 +161,23 @@ M0_INTERNAL void m0_fid_tgenerate(struct m0_fid *fid,
 	m0_fid_tchange(fid, tid);
 }
 
+M0_INTERNAL bool m0_base_fid_eq(const struct m0_fid *fid0,
+				const struct m0_fid *fid1)
+{
+	/*
+	 * This function is to compare two FID's with base part of the FID's.
+	 * During comparison it will skip counter bits (higher 32 bits
+	 * of key) to compare two FID's.
+	 */
+	if ((fid0->f_container == fid1->f_container) &&
```

Comment:
return (fid0->f_container == fid1->f_container) &&
	           ((fid0->f_key & M0_FID_DYNAMIC_CNT_MASK) ==
	           (fid1->f_key & M0_FID_DYNAMIC_CNT_MASK));
	           
	  I  think brackets too are not needed.
	return fid0->f_container == fid1->f_container ?:
	           fid0->f_key & M0_FID_DYNAMIC_CNT_MASK ==
	           fid1->f_key & M0_FID_DYNAMIC_CNT_MASK;